client/src/components/molecules/FilterSection/FilterSection.types.tsx

```diff
@@ -0,0 +1,6 @@
+export default interface FilterSectionPropType {
+  mappingArray: {displayText: string, valueText: string}[];
+  elementName: string;
+  filterValue?: undefined | string;
```

Comment:
i've put it in but as optional as I think I might take it out since filterValue isn't actually used in the component - only setFilterValue - wasn't really sure though if it was okay to only pass the setting component of a useState so put it in as an optional to placehold and remind me